packages/augur-sdk/src/state/getter/Universe.ts

```diff
@@ -13,8 +18,25 @@ export interface DisputeWindow {
   fees: string;
 }
 
+export interface NonForkingMigrationTotals {}
+export interface MigrationTotals {
+  marketId: string;
+  outcomes: Outcome[];
+}
+
+interface Outcome {
+  outcomeName: string;
+  outcome: string; // non-scalar markets this is outcome id.
+  amount: string; // atto-rep given to this outcome
+  isMalformed: boolean;
+  payoutNumerators: string[]; // not needed by UI, but 3rd parties might want it
+}
```

Comment:
I missed we need a flag `isInvalid` for scalars when outcome is 0. 
0 outcome id can be either a price between min and max or invalid

Reply:
Can I just use `isMalformed`?

Reply:
isMalformed isn't the same as isInvalid

Reply:
NEVERMIND!